MetamodelAlgemeen.md

```diff
@@ -307,18 +307,20 @@ Een object heeft kenmerken waarvoor gegevens bekend zijn.
+Een model specificeert niet de gegevens zelf. Een gegeven zoals '1-1-1970' noemen we een waarde voor een attribuut van Jan. In het informatiemodel wordt dit bv. het attribuutsoort 'geboortedatum' of attribuutype 'overlijdensdatum' van een objecttype Persoon genoemd. Het is dus ook van belang om welke eigenschap van welke specifieke persoon het gaat. Dat twee objecten een relatie hebben wordt ook gezien als een een modelelement waarvoor een gegeven bekend kan zijn. 
 
-Een gegeven kan al bekend zijn zodra deze wordt waargenomen, maar in een informatievoorziening zijn gegevens de objectief waarneembare neerslag of registratie van feiten op een bepaald medium, zodanig dat deze gegevens uitgewisseld en voor langere tijd bewaard kunnen worden. Dat kan op papier, in digitale vorm, et cetera. Een gegeven in een informatievoorziening is dus een vastgelegde waarde voor een waargenomen feit dat toegekend is aan een kenmerk van een specifiek object. 
+Bewering: een uitspraak, die meestal als waar bedoeld is maar niet per definitie waar is. Mijn geboortedatum is '1-1-1970', de waarde van dit huis is 200.000 euro (in het jaar 2018), de sensor waarneming dat de auto met kenteken 'AA 1234' met een snelheid van 120 kilometer per uur reed op de A1 bij hectometeraanduiding 20.8 op tijdsmoment 16:00. Wie de bewering gedaan heeft is vaak relevant. Denk bij een bewering aan hetgeen waarover besloten wordt door een bevoegd gezag, of aan een persoon die gegevens opgeeft voor een bepaald doel. 
 
-Er zijn waardes die:
+Vastgelegd: een gegeven kan al bekend zijn zodra deze wordt waargenomen, maar in een informatievoorziening zijn gegevens de objectief waarneembare neerslag of registratie van feiten op een bepaald medium, zodanig dat deze gegevens uitgewisseld en voor langere tijd bewaard kunnen worden. Dat kan op papier, in digitale vorm, et cetera. Een gegeven in een informatievoorziening is dus een vastgelegde _waarde_ van een waarneming om bewering over een specifieke eigenschap van een specifiek object. 
+
+De modelelementen waarmee een getypeerde eigenschap van een object gemodelleerd kan worden zijn: _attribuutsoort_, _relatiesoort_ en _externe koppeling_. Hier kunnen gegevens over bekend zijn. Het datatype is hierbij van belang voor (alleen) de waarde, en het objecttype is van belang omdat het helder moet zijn om welk object het gaat.
```

Comment:
Strict genomen hebben we iets breders nodig dan Datatype op de waarde aan te duiden, omdat in MIM een objecttype geen Datatype is. Bijvoorbeeld Waardetype?

Reply:
Hopelijk is een objecttype in geen enkele modelleertaal ooit een datatype (!), dan gaat er iets verschrikkelijk mis.

Ik vermoed dat we iets nodig hebben om een gegeven eenduidig in te kunnen structureren. Zoals jij altijd aangeeft, onderwerp, eigenschap, waarde en ik denk dan ook: waarheidswaarde. Dit is echter voer voor 2.0